### PR TITLE
Fix freebsd fuse tests

### PR DESCRIPTION
**Pull request by ThomasWaldmann**

### CODE REVIEW

File: borg/testsuite/archiver.py [hunk 1]
```diff
@@ -1097,6 +1088,27 @@ def has_noatime(some_file):
...
+                    assert xattr.getxattr(out_fn, 'user.foo') == b'bar'
+                else:
+                    assert xattr.listxattr(out_fn) == []
+                    try:
+                        xattr.getxattr(out_fn, 'user.foo')
+                    except OSError as e:
+                        assert e.errno == llfuse.ENOATTR
+                    else:
+                        assert False, "expected OSError(ENOATTR), but no error was raised"
+            except OSError as err:
```
Comment (ThomasWaldmann) [inline borg/testsuite/archiver.py]: i just moved this "list/read xattrs" block to the end and added try/except to ignore the freebsd issue.

**Comment by ThomasWaldmann** [comment]
merge as is?

**Comment by enkore** [comment]
I guess. I looked at it earlier today, I obviously can't judge by looking at code whether that actually fixes the issue, but you probably checked that :)

**Comment by ThomasWaldmann** [comment]
yes, it worked. the only questionable thing is whether this should work on freebsd or not, but i couldn't find much about that.

**Pull request merged**